### ISSUE TITLE
Project is alive?

### ISSUE THREAD

**Issue by Argentum88** [leo-project/leofs#1197]
I noticed that in late April, active development was suspended. What happened?

**Comment by mocchira** [comment]
@Argentum88 Sorry for the late reply.

Originally @yosukehara project lead should answer your question but he seems busy so I'd like to answer instead of him.

Now he and other project members (including me) have been involved in other projects with more high priority so LeoFS has been not active these days.

One thing I can say at this moment is we will keep doing the maintenance release for LeoFS whenever we find any bugs which should be fixed.

@yosukehara please fix me if there are something wrong on the above comment.

**Comment by yosukehara** [comment]
@Argentum88 
Thank you for your comment.
The answer to your question is as @mocchira wrote.

**Comment by wulfgar93** [comment]
@yosukehara @mocchira could you tell when you plan to return to the development of LeoFS?

**Comment by yosukehara** [comment]
> @yosukehara @mocchira could you tell when you plan to return to the development of LeoFS?

Thank you for your comment.
I'm involved in another project from last year. So I cannot take the time to develop LeoFS.
I'm going to consider organizing my work and making LeoFS development time.

**Comment by mocchira** [comment]
> @yosukehara @mocchira could you tell when you plan to return to the development of LeoFS?

@wulfgar93 The situation I've been in is almost same as @yosukehara. I'm not sure when I can come back to LeoFS developement for now however I'll be back ASAP.

**Comment by dumblob** [comment]
I was recently looking for a storage system with properties of LeoFS and wonder whether it's a good idea to use LeoFS in production considering it's not backend by any commercial interest any more.

Any new insights?

**Comment by nkosi23** [comment]
I am interested in knowing if LeoFS would be recommended for new projects as well

**Comment by dyasny** [comment]
+1

**Comment by nkosi23** [comment]
@dyasny In the meantime I have found out that seaweedfs is the established and recommended alternative. Hope this helps

**Comment by dyasny** [comment]
@nkosi23 yeah, I'm aware of it. Unfortunately, it doesn't support NFS, and that's what I need for now

**Comment by onlyjob** [comment]
> @nkosi23 yeah, I'm aware of it. Unfortunately, it doesn't support NFS, and that's what I need for now

Maybe [Ganesha](https://github.com/nfs-ganesha/nfs-ganesha) can be of help? It is an NFS server in user space that can present Fuse file systems over NFS.

SeaweedFS is not great with POSIX compliance/compatibility...

I wholeheartedly recommend [MooseFS](https://github.com/moosefs/moosefs) as alternative.

**Comment by dyasny** [comment]
> Maybe [Ganesha](https://github.com/nfs-ganesha/nfs-ganesha) can be of help? It is an NFS server in user space that can present Fuse file systems over NFS.

I can simply set up a basic NFS server, I think performance would actually be better. But What I actually need is a resilient NFS service, distributed, replicated, the works. I am aware of ceph and gluster of course and they are an overkill for me. 



> SeaweedFS is not great with POSIX compliance/compatibility...
> 
> I wholeheartedly recommend [MooseFS](https://github.com/moosefs/moosefs) as alternative.

Frankly, I don't care too much about posix compliance. What I need to do is mount the same NFS export on all of my k8s workers and use that mountpoint for hostPath volumes. Inside the volumes there's an app with a few sidecars and an sqlite file.

**Comment by mihai-satmarean** [comment]
@dyasny have you looked at ZFS?
